StripeUICore/StripeUICore/Source/Elements/Factories/Address/AddressSectionElement.swift

```diff
@@ -335,7 +350,12 @@ import UIKit
             }
         }
 
-        let initialElements: [Element?] = [name, country, autoCompleteLine]
+        var initialElements = [Element?]()
+        initialElements.append(name)
+        if collectionMode != .noCountry {
+            initialElements.append(country)
+        }
+        initialElements.append(autoCompleteLine)
```

Comment:
```suggestion
        var initialElements = [name]
        if collectionMode != .noCountry {
            initialElements.append(country)
        }
        initialElements.append(autoCompleteLine)
```

Reply:
I was gonna do it this way, but since I needed to specify the type explicitly and would need to append anyway I went the other way. Changed.